common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1545,7 +1545,10 @@ public enum PUBLIC_SHARE_VISIBILITY { samePrimaryDomain, all, none };
 
     // ZBUG-3105: zimbra_allowed_redirect_url is a url that allows in preauth redirectURL
     public static final KnownKey zimbra_allowed_redirect_url = KnownKey.newKey("");
-    
+
+	// ZCS-12948: ssdb_zok_compat is true if we should write ephemeral values as JSON strings.
+	public static final KnownKey ssdb_zok_compat = KnownKey.newKey(false);
```

Comment:
I think we should rename this LC attr as `ssdb_zimbrax_compatible` instead of using zok

Reply:
Done.